bundles/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/internal/handler/TeslaAccountHandler.java

```diff
@@ -308,7 +308,8 @@ private ThingStatusInfo authenticate() {
         return new ThingStatusInfo(ThingStatus.ONLINE, ThingStatusDetail.NONE, null);
     }
 
-    protected String invokeAndParse(String vehicleId, String command, String payLoad, WebTarget target) {
+    protected String invokeAndParse(String vehicleId, String command, String payLoad, WebTarget target,
+            int noOfretries) {
```

Comment:
Typo? nbOfretries?

Reply:
_No._ is the correct abbreviation for _Number_. So noOf**R**etries or **number**Of**R**etries for perfection. :)

Reply:
In French, the abbreviation for number is nb, I thought it was the same in English.

Reply:
See https://en.wikipedia.org/wiki/Numero_sign